apps/chat/channels.py

```diff
@@ -217,7 +227,7 @@ def _handle_pre_conversation_requirements(self):
         (Status==PENDING_PRE_SURVEY) user indicated that they took the survey -> sett status to ACTIVE
         """
         # We manually add the message to the history here, since this doesn't follow the normal flow
-        self._add_message_to_history(self.message_text, ChatMessageType.HUMAN)
```

Comment:
Is `message_text` used anywhere now? Should all references be switched?

Reply:
yes, `message_text` is only used inside the `_extract_user_query` method

Reply:
Can we rename it to `get_message_text` to be symmetrical with `get_message_audio` (and make it a method).

Side note: it seems like a lot of the subclasses of `ChannelBase` have the same implementation for some of these functions like `message_content_type`. Feels like we could make it cleaner. Maybe there should be a base message type which exposes a consistent interface rather that having the channel manage it?